CMakeLists.txt

```diff
@@ -29,6 +29,11 @@ find_package(NetCDF REQUIRED Fortran)
 
 add_subdirectory(src)
 
+include(CTest)
+if(BUILD_TESTING)
+    add_subdirectory(tests)
+endif()
```

Comment:
why is this in a if-block? It should be built regardless.  Running is optional.

Reply:
That's the variable CTest uses. It's on by default and is what's used in their documentation.

https://cmake.org/cmake/help/latest/module/CTest.html